deeppavlov/dataset_iterators/file_paths_iterator.py

```diff
@@ -74,6 +70,6 @@ def gen_batches(self, batch_size: int, data_type: str = 'train', shuffle: bool =
         for shard in shard_generator:
             if not (batch_size):
                 bs = len(shard)
-            lines_generator = self._chunk_generator(shard, bs)
+            lines_generator = chunk_generator(shard, bs)
```

Comment:
`bs` is not difined if `batch_size` is specified

Reply:
tnx